### PR TITLE
Fixed one of the To-be-done sections

### PR DESCRIPTION
**Pull request by xml-project**

### CODE REVIEW

File: steps/src/main/xml/steps/http-request.xml [hunk 1]
```diff
@@ -230,10 +230,10 @@ For those methods the documents on the <port>source</port> are ignored when cons
           <varlistentry>
             <term>transfer-encoding</term>
             <listitem>
-              <note xml:id="http-uri-transfer-encoding" role="editorial">
-                <title>Editorial Note</title>
-                <para>TBD: How does this entry differ from the "transfer-encoding" header?</para>
-              </note>
+              <para>The value associated with the “<literal>transfer-encoding</literal>” key must be an instance
+                of <code>xs:string</code>. If the key-value pair is present, the response will be treated as if the response
+                contained a “<literal>Transfer-Encoding</literal>” header given by its value. The original Transfer-Encoding
```
Comment (gimsieke) [inline steps/src/main/xml/steps/http-request.xml]: Suggestion: contained a “<literal>Transfer-Encoding</literal>” header with the same value

Also for `Content-Encoding` in the preceding item.

File: steps/src/main/xml/steps/http-request.xml [hunk 2]
```diff
@@ -230,10 +230,10 @@ For those methods the documents on the <port>source</port> are ignored when cons
...
+                will however be reflected unchanged as entry in the document on the <port>report</port> port.</para>
```
Comment (gimsieke) [inline steps/src/main/xml/steps/http-request.xml]: The original Transfer-Encoding will however be reflected unchanged as entry in the document on the report port. 
→
The original Transfer-Encoding *header value* will however be reflected unchanged as *an* entry in the document on the report port. 

Btw, we didn’t mention the `report` port in the spec. Instead, there is `response` port in “The response's status code and the response's header fields will be made available on the response port as a map.”. It should probably be changed into “`report` port”.

Comment (xml-project) [reply steps/src/main/xml/steps/http-request.xml]: The port "report" is named in the step's signature. But the line you found is surely a typo.

**Comment by xml-project** [comment]
Since the changes requested by @gimsieke also apply to at least another sections (where parameter 'content-encoding') I propose you discuss this on Thursday's call and leave me a note about your resolution.

File: steps/src/main/xml/steps/http-request.xml (hunk 2 above)
Comment (xml-project) [reply steps/src/main/xml/steps/http-request.xml]: Typo in port name addressed in pr #341

Comment (xml-project) [reply steps/src/main/xml/steps/http-request.xml]: ... and solved.

**Comment by xml-project** [comment]
Overtaken by pr #346

**Pull request closed without merging**